pkg/kubelet/cm/container_manager_linux.go

```diff
@@ -968,15 +964,18 @@ func (cm *containerManagerImpl) GetDynamicResources(pod *v1.Pod, container *v1.C
 		// a set of CDIDevices from a different kubelet plugin. In the future we may want to
 		// include the name of the kubelet plugin and/or other types of resources that are
 		// not CDIDevices (assuming the DRAmanager supports this).
-		for _, klPluginCdiDevices := range containerClaimInfo.CDIDevices {
+		for /* driverName */ _, driverState := range containerClaimInfo.Drivers {
```

Comment:
Is commenting like this something we are doing now when iterating over maps and we want to ignore the key?

Reply:
I'm not sure whether it is a common practice. I find it useful as reminder of what it is that is not being used.

Reply:
OK. I'm not opposed to it -- I just haven't seen anyone do it before (at least not in this code base).